elephant/statistics.py

```diff
@@ -1429,6 +1430,8 @@ def _epoch_with_spread(self):
         # Note: all spike trains share the same t_start and t_stop.
         left_edges[0] = max(self.t_start, left_edges[0])
 
+        complexities = complexities.astype(np.uint16)
```

Comment:
This seems redundant with line 1417. Can one not directly initialize complexities as `np.uint16`?

Reply:
I do not remember exactly why included this line. I am guessing that after doing some operations in the array it turned the values to a more memory consuming type. @Kleinjohann do you remember if that was the case?

Reply:
@mdenker is right, I changed it to immediately cast to `np.unit16`.